typescript/web/src/components/auth-manager/signin-modal/__stories__/signin-modal.tsx

```diff
@@ -30,3 +30,15 @@ export const LinkSent = () => {
     />
   );
 };
+
+export const ErrorSigninMethod = () => {
+  return (
+    <SigninModal
+      isOpen
+      error="OAuthAccountNotLinked"
```

Comment:
Can't this value be retrieved from one of next-auth constants?

Reply:
It does not look like it, as the error in the `SignInResponse` interface is simply a `string`